packages/commerce-sdk-react/src/components/StorefrontPreview/utils.ts

```diff
@@ -22,7 +23,7 @@ export const detectStorefrontPreview = () => {
 export const getClientScript = () => {
     const parentOrigin = getParentOrigin() ?? 'https://runtime.commercecloud.com'
     return parentOrigin === DEVELOPMENT_ORIGIN
-        ? `${parentOrigin}/mobify/bundle/development/static/storefront-preview.js`
+        ? `${parentOrigin}${LOCAL_BUNDLE_PATH}/static/storefront-preview.js`
         : `${parentOrigin}/cc/b2c/preview/preview.client.js`
```

Comment:
The `/mobify/bundle/development` path here is only relevant to us when we are testing storefront preview against a locally deployed Runtime Admin. 

But we might want to namespace `/cc/b2c/preview/` as well?

Reply:
The `/cc/b2c/preview` is handled by MRT directly. It is not a route modifiable by clients. Customers will never need to set a namespace for it.